### DISCUSSION TITLE
Understanding the data of the "Extent" field in SKSEC -  Manuscript

### DISCUSSION THREAD

**Discussion by AmirSimantov** [PanditProject/Website#8] [Help] [answered]
This discussion is relevant to a previous one, but it isn't the same. See #5 

### CONTEXT
Hi. I need your help here. A few years ago, we imported the SKSEC database into PanditProject.
One of the fields was "Extent."

### NEED
The "Extent" field is now deprecated. We want to retrieve each piece of data from it to the correct specific field.

### PROBLEM
As the field was a free text one, I am unsure which piece of data should go into which field.

### RESOURCE
[(Extent tab) - Manuscript -> Size, Extent and related fields ](https://docs.google.com/spreadsheets/d/1PXeNynTj8ZrrXEXf2ni19nEBkIyfCTOjoBvnVwioLZU/edit#gid=1048840650)
Some of the columns check if a condition is met; the value 1 for true and the value 0 for false). You may want to sort by such a column to look for positives (Right-click a column and select "Sort sheet Z->A").

### Our relevant fields:
The first three fields are simple, while the other two are inner fields inside a complex field.
For the complete list of all the fields, see: [(Manuscript tab) - Configuration spreadsheet](https://docs.google.com/spreadsheets/d/1SLjJyuPEU8BXfschex56HGRAA9M3AKjMWWlyq7cJf88/edit#gid=97)

1. Original number of folios
2. Extant number of folios
3. Extant folio range
4. Work contained -> Range of folios
5. Work contained -> Extant portions

A value for the field "Completeness" may or may not be inferred.

### The examples
Here are some cases that seem to cover the patterns that I have found.
I have grouped them according to visual similarity, not a semantic one. Nevertheless, you will probably tell me that some of the patterns reveal the same semantics.
**Any help will be appreciated.**
In your comments, please refer to an example by copying it.

1 Folio
2 Folios

1 Folio (47th)
1 Folio (12th)

Folios 99a - 101a
Folios 1a-7b

Folios 1;3-26; 2; 2-3; 13
Folios 1-3; 1-3; 5-7; 1-4, etc.

Folios 1-3; 3-6 [sic]

Folio 1 68+9

Folios 1-231 = (232-241) 242
Folios 1-183; 185-434; 436-486; 488-507 + 2

7 ff
1 - 58 ff

Ff. 1-7 (library)
Ff. 1-4 (library); ff. 1,5,5,13 (scribe)
Ff. 1-24 (library); ff. 2-[25] (scribe)

61 ff (kāraka section)
36 ff. (includes ch. 12 on śaurasenī)
8 ff. (comm. on gajastra only; another copy)
16ff. (another copy)
50ff. (adhyāyas 2, 7 and part of 8). With kātyāyana's vārttikas and the śabdabhūṣaṇa of nārāyaṇa sudhī.
4ff (ends with commentary on P.3.1.134)

**Comment by AmirSimantov** [comment]
@ybronner, please see if you can answer part of it. Please also mention any collaborator that may help us with it. Thanks.

**Reply by ybronner** [reply]
I'll try to focus on two aspects of this complex question. First, how to get information on the work contained (fields 4 and 5) in the relevant manuscripts from the above list. I think the majority of the manuscripts only contain one work, so in all of these cases, if we can separate them, there should not be a problem. Does this make sense? We will still have to work on the manuscripts that contain more than one work.
Second, the notes in the examples such as this: 
50ff. (adhyāyas 2, 7 and part of 8). With kātyāyana's vārttikas and the śabdabhūṣaṇa of nārāyaṇa sudhī.
The information in the note tells you two things: 
1. which chapters are included of this main work (2,7, and a part of 8; adhyāya is a word for chapter). 
2. Which commentaries accompany this work in the manuscript. The problem: I’m not sure we should treat this as a manuscript that contains one work and not three, namely, the main text of Panini, the Vārttikas, and the Śabdānuśana. I’m not sure we were very consistent in separating the work from the commentaries in connecting manuscripts to works. This might be a whole can of worms…

**Reply by ybronner** [reply]
@aso2101 Any thoughts?

**Comment by ybronner** [comment]
@wujastyk Let me know if you have any thoughts

**Comment by wujastyk** [comment]
I'm sorry, I don't have enough time to give a proper answer.  But
cataloguers of western manuscripts back away from forcing such decisions by
downgrading, so to speak, the concept of "work".  A manuscript is described
as a physical object that has writing on it.  On pages 1-10 there is some
Aristotle.  On page 11 there is a prayer beginning "our father".  On pages
12-33 there is Hamlet by Shakespeare, that breaks off at act III.  And so
on.  The way a user finds things is through indexes that are done like a
book index.

PP can do all this.  I think maybe the can of worms arises when we create
too many categories.

I would raise the question, too, of whether PP is turning into a
cataloguing tool.  I am not sure that is the best direction.  Who is the
audience?  Perhaps a primary audience is editors who which to locate the
MSS of a work so that they can gather them for editing work.  They will do
their own MS descriptions.  What they mainly need is to locate "Aristotle"
"Hamlet" etc.  Another audience is genealogists or intellectual historians
who need family trees of scholars.  And so on.  Maybe thinking about the
audiences can help with the design of the data?

In the example you give, Nārāyaṇasudhī should be indexed.  Also his work
Śabdānuśāsana.  This is apparently an unknown work (not in NCC)

Sorry, again.  I'm sure you know all this already and I haven't precisely
answered your question.

Best,
Dominik

**Comment by ybronner** [comment]
@wujastyk: I want to narrow down this complex conversation to just one aspect of interpreting the data about the extent of manuscripts. Below I list the samples Amir gathered and, when I think I know what they mean, I interpret them accordingly. When I don't, I leave a question mark. Can I ask you to tell me if I am correct in my interpretations, and to suggest yours where I left a question mark? If needed, you can look up the [resource](https://docs.google.com/spreadsheets/d/1PXeNynTj8ZrrXEXf2ni19nEBkIyfCTOjoBvnVwioLZU/edit#gid=1048840650) Amir has created.

1 Folio
2 Folios
**Yigal: I think this means we have just 1 and 2 folios respectively: this should go in "Extant number of folios"**

1 Folio (47th)
1 Folio (12th)
**Yigal: ? I don't know what "(47th)" and "(12th)" mean. I assume this 1 goes to "Extant number of folios"**

Folios 99a - 101a
Folios 1a-7b
**Yigal: I take this to mean that this is the "Extant folio range"**

Folios 1;3-26; 2; 2-3; 13
Folios 1-3; 1-3; 5-7; 1-4, etc.
**Yigal: ? I take this to also mean that this is the "Extant folio range." But I don't know what the semicolons mean, why the ranges overlap, and what the "etc." means**

Folios 1-3; 3-6 [sic]
**Yigal: ? what mistake does the "[sic]" refer to? The overlap?**

Folio 1 68+9
**Yigal: ? Not sure what the range is and what the + sign refers to**

Folios 1-231 = (232-241) 242
**Yigal: ? Not sure what the = sign and the parenthetical numbers refer to**

Folios 1-183; 185-434; 436-486; 488-507 + 2
**Yigal: I assume this means "Extant folio range" and the "+2" refers to two unnumberd folios**

7 ff
1 - 58 ff
**Yigal: I assume that "ff" means "folios, and that "7ff" should go in "Extant number of folios"; and "1 - 58 ff," should go into "Extant folio range"**

Ff. 1-7 (library)
Ff. 1-4 (library); ff. 1,5,5,13 (scribe)
Ff. 1-24 (library); ff. 2-[25] (scribe)
**Yigal: ? I don't understand the use of "library" and "scribe" in this context**

**Comment by wujastyk** [comment]
On Mon, 7 Jun 2021 at 09:12, Yigal Bronner <notifications@github.com> wrote:

    @wujastyk: I want to narrow down this complex conversation to just one aspect of interpreting the data about the extent of manuscripts. Below I list the samples Amir gathered and, when I think I know what they mean, I interpret them accordingly. When I don't, I leave a question mark. Can I ask you to tell me if I am correct in my interpretations, and to suggest yours where I left a question mark? If needed, you can look up the resource Amir has created.

    1 Folio
    2 Folios
    Yigal: I think this means we have just 1 and 2 folios respectively: this should go in "Extant number of folios"

Yes

 

    1 Folio (47th)
    1 Folio (12th)
    Yigal: ? I don't know what "(47th)" and "(12th)" mean. I assume this 1 goes to "Extant number of folios"


This means that we have two folios, one with the number 47 written on it and another with the number 12 written on it.

    Folios 99a - 101a
    Folios 1a-7b
    Yigal: I take this to mean that this is the "Extant folio range"


yes.  I don't know what the a and  b mean.

    Folios 1;3-26; 2; 2-3; 13
    Folios 1-3; 1-3; 5-7; 1-4, etc.
    Yigal: ? I take this to also mean that this is the "Extant folio range." But I don't know what the semicolons mean, why the ranges overlap, and what the "etc." means


These numbers are the scribal numbers written on the folios.  So in the first example, we have folios with the following numbers written on them:

1, 3, 4, 5, 6, 7,8 9, 10 ... 23, 24, 25, 26, 2, 2, 3, 13.


 

    Folios 1-3; 3-6 [sic]
    Yigal: ? what mistake does the "[sic]" refer to? The overlap?


We have folios with this foliation: 1, 2, 3, 3, 4, 5, 6. 

The "sic" means that yes, there are two folios numbered 3.

    Folio 1 68+9
    Yigal: ? Not sure what the range is and what the + sign refers to


I don't know either.  Most likely a single folio that is numbered both 68 and 69 simultaneously; but I'm not sure.

    Folios 1-231 = (232-241) 242
    Yigal: ? Not sure what the = sign and the parenthetical numbers refer to

I don't know either.

 

    Folios 1-183; 185-434; 436-486; 488-507 + 2
    Yigal: I assume this means "Extant folio range" and the "+2" refers to two unnumberd folios

Yes.  These are again numbered folios, but missing folios 184, 435, and 487.  At the end there are two unnumbered folios.

 

    7 ff
    1 - 58 ff
    Yigal: I assume that "ff" means "folios, and that "7ff" should go in "Extant number of folios"; and "1 - 58 ff," should go into "Extant folio range"


Yes, ff means there are seven folios. 

    Ff. 1-7 (library)
    Ff. 1-4 (library); ff. 1,5,5,13 (scribe)
    Ff. 1-24 (library); ff. 2-[25] (scribe)
    Yigal: ? I don't understand the use of "library" and "scribe" in this context

When a scribe first copies out a MS he usually numbers the leaves.  Indian scribes can be sloppy about this, so the sequences are often imperfect.

When a MS arrives at a (professional) library, the MS goes through a process of "accessioning" which means that a brief record is made of what has just been received.  This process should include "foliation" which means that a qualified librarian will go through the MS and write a small pencil number on each leaf, numbering from 1 to the end.  These "library foliations" are a continuous sequence that ignores the scribe's original numbering.  The canonical way to refer to the parts of a MS in a major library that uses these practices is by these library foliations (not by the scribe's numbers).  Most indologists don't know this or follow this practice.


P. 61 of Greg Pass's old Library of Congress guide, AMREMM, gives examples of some complex layouts.


I hope this helps!

**Reply by ybronner** [reply]
It helps a lot! Many thanks, Dominik. I'll write a more detailed reply in a bit, but I think this solves most of these data queries and helps locate them in the relevant fields.

**Reply by AmirSimantov** [reply]
@wujastyk 
When there _isn't_ a "(library)" comment, can we say _for sure_ that the numbers are the **scribal** page numbers?

**Reply by AmirSimantov** [reply]
[Dominik answered below]

**Comment by wujastyk** [comment]
Well, in this field nothing is ever completely certain :-).  But yes, the
folio numbers one sees in catalogues and in Indologist's publications are
*commonly* the scribal ones.  The exception is in the big, professional or
national libraries like the Bodleian, the British Library, Bibliotheque
Nationale etc. where they do things "properly" and use their own library
foliations, especially in their official publications.

In situations like this, maybe we are trying to make PP too precise?
Rather than trying to nail things down ever more precisely, should we back
off and have just a note field or a single field saying "roughly how many
folia are there?"   Or maybe PP shouldn't have anything about foliation and
folia at all?  I'm serious.  Why are we putting that info into the system
in the first place?  Are we really interested?  Will our users expect to
find the foliation information in PP?  Wouldn't it be better to have robust
pointers to the catalogues, and let the users find the information there?
Now that more and more catalogues are available as scans, that is becoming
a realistic solution.

Best,
Dominik

**Reply by AmirSimantov** [reply]
[
Dominik answered my question [here](https://github.com/PanditProject/Website/discussions/8#discussioncomment-945046):
When there _isn't_ a "(library)" comment, can we say _for sure_ that the numbers are the **scribal** page numbers?
]

Thank you, @wujastyk.

Regarding Pandit's scope, the question is more relevant to fields that we _don't_ have yet and consider having, while the fields regarding foliage already exist. And, they don't exist by chance; in the deprecated "Extent" field, we had put foliage values when we imported the SKSEC database (for more than a thousand entities). Then, @SukshmadarshiMaharaj used it for folios/leaves (again, more than a thousand entities). I doubt that you-all historians would want to throw all this data away...

Regarding your important input about the numbers, I understand that we can go on with the _de facto_ writing (and not the much less common _comme il faux,_ writing) that is, unless noted differently, the numbers are the scribal ones. So, the help text for the "Extant folio ranges" field will include a sentence such as:
Normally, enter the scribal's page numbers. If the number isn't the scribal's ones, state in parentheses who added them.  <em>Example</em> <small>3-12 (library)</small> 

Does it make sense?

**Reply by wujastyk** [reply]
Yes.

**Comment by AmirSimantov** [comment, marked as answer]
The previously deprecated "Extent" field is now obsolete.
Where possible, we moved the value to more specific fields, otherwise, to the "Discrete facts" field.

Thanks to all of you who helped - here and in private emails :)

A revision note was added to each change: Values moved from the obsolete "Extent" field.

**Total entities with a value in the obsolete field: 5,520** (5,243 changed with a script and 277 manually).
